Tests/ApodiniTests/ConfigurationTests/HTTPConfigurationTests.swift

```diff
@@ -0,0 +1,46 @@
+@testable import Apodini
+import XCTest
+
+final class HTTPConfigurationTests: ApodiniTests {
```

Comment:
I think it would make sense to have a tests that checks if command line arguments can overwrite in-code configurations.

Reply:
good point! done.